src/Components/ServiceScene/Breakdowns/ClearFiltersLayoutScene.tsx

```diff
@@ -0,0 +1,24 @@
+import { SceneComponentProps, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
```

Comment:
Maybe rename to `NoResultsScene` or `NoMatchingLabelsScene`? Also, does it need "Layout" as part of the name?